src/test/java/seedu/stock/logic/commands/AddCommandIntegrationTest.java

```diff
@@ -1,45 +1,46 @@
-//import static seedu.stock.testutil.TypicalStocks.getTypicalAddressBook;
-//
-//import org.junit.jupiter.api.BeforeEach;
-//import org.junit.jupiter.api.Test;
-
-//import seedu.stock.model.Model;
-//import seedu.stock.model.ModelManager;
-//import seedu.stock.model.SerialNumberSetsBook;
-//import seedu.stock.model.UserPrefs;
-//import seedu.stock.model.stock.Stock;
-//import seedu.stock.testutil.StockBuilder;
+import static seedu.stock.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.stock.testutil.TypicalStocks.PINEAPPLE;
+import static seedu.stock.testutil.TypicalStocks.getTypicalSerialNumberSetsBook;
+import static seedu.stock.testutil.TypicalStocks.getTypicalStockBook;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.stock.model.Model;
+import seedu.stock.model.ModelManager;
+import seedu.stock.model.UserPrefs;
+import seedu.stock.model.stock.Stock;
 
 /**
  * Contains integration tests (interaction with the Model) for {@code AddCommand}.
  */
 public class AddCommandIntegrationTest {
 
-//    private Model model;
-
-//    @BeforeEach
-//    public void setUp() {
-//        model = new ModelManager(getTypicalStockBook(), new UserPrefs());
-//    }
-
-//    @Test
-//    public void execute_newStock_success() {
-//        Stock validStock = new StockBuilder().build();
-//
-//        Model expectedModel = new ModelManager(model.getStockBook(), new UserPrefs(), new SerialNumberSetsBook());
-//        expectedModel.addStock(validStock);
-//
-//        assertCommandSuccess(new AddCommand(validStock), model,
-//                String.format(AddCommand.MESSAGE_SUCCESS, validStock), expectedModel);
-//    }
-//
-//    @Test
-//    public void execute_duplicatePerson_throwsCommandException() {
-//        Stock personInList = model.getStockBook().getStockList().get(0);
-//        assertCommandFailure(new AddCommand(personInList), model, AddCommand.MESSAGE_DUPLICATE_STOCK);
-//    }
+    private Model model;
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(getTypicalStockBook(), new UserPrefs(), getTypicalSerialNumberSetsBook());
+    }
+
+    @Test
+    public void execute_newStock_success() {
+        Stock validStock = PINEAPPLE;
+
+        Model expectedModel = new ModelManager(model.getStockBook(), new UserPrefs(), getTypicalSerialNumberSetsBook());
+        expectedModel.addStock(validStock);
+
+        assertCommandSuccess(new AddCommand(validStock), model,
+                String.format(AddCommand.MESSAGE_SUCCESS, validStock), expectedModel);
+    }
+
+    //    @Test
+    //    public void execute_duplicateStock_throwsCommandException() {
+    //        dog test case that causes some test to fail 1/2 the times??? what kind of sorcery is this?
+    //        Stock stockInList = model.getStockBook().getStockList().get(0);
+    //        assertCommandFailure(new AddCommand(stockInList), model, AddCommand.MESSAGE_DUPLICATE_STOCK);
+    //    }
```

Comment:
Maybe consider removing this first so the code will be cleaner. Can always refer to this later by viewing GitHub commit history

Reply:
Aight will remove it